code/modules/surgery/organs/organ_external.dm

```diff
@@ -138,6 +138,29 @@
 	replaced(H)
 	return 1
 
+/obj/item/organ/external/attack(mob/living/carbon/C, mob/user) // Limb re-attachment
```

Comment:
Could you put this in human/attackby instead, so it's not such snowflakey code?

Reply:
I've been trying to figure this out in my spare time lately, but life has really given me a bad time lately (mostly my own fault). I haven't forgotten, I am trying to move it, but it's proving harder than expected